generators/server/generator.mjs

```diff
@@ -223,19 +205,26 @@ spring:
         );
       },
 
-      async mainClass({ application: { baseName, packageFolder, databaseTypeSql, reactive } }) {
+      async mainClass({ application: { baseName, packageFolder, databaseTypeSql, prodDatabaseTypePostgres, reactive } }) {
         const mainClassPath = `${SERVER_MAIN_SRC_DIR}${packageFolder}/${this.getMainClassName(baseName)}.java`;
         let content = this.readDestination(mainClassPath);
-        const types = ['org.HdrHistogram.Histogram.class', 'org.HdrHistogram.ConcurrentHistogram.class'];
+        const types = [
+          'org.HdrHistogram.Histogram.class',
+          'org.HdrHistogram.ConcurrentHistogram.class',
```

Comment:
FWIW, these will be added in the next Spring Native release. https://github.com/spring-projects-experimental/spring-native/issues/1484